doc/builders/packages/shell-helpers.xml

```diff
@@ -16,6 +16,16 @@
      <literal>fzf</literal>: <command>fzf-share</command>
     </para>
    </listitem>
+   <listitem>
+    <para>
+     <literal>zsh-autoenv</literal>: <command>zsh-autoenv-share</command>
+    </para>
+   </listitem>
```

Comment:
Did you intend to include zsh-autoenv in this commit?

Reply:
Yes, I saw that this was missing as well. It even seemed like fzf was one of the first like this and zsh-autoenv copied from this.